### ISSUE TITLE
Python string.strip() optional arguments not implemented

### ISSUE THREAD

**Issue by shaolo1** [TranscryptOrg/Transcrypt#41]
Python

```
foo = 'foocba'
print(foo.strip('abc'))
```

foo
Generated javascript prints
foocba

**Comment by JdeH** [comment]
Indeed this was not yet done to keep the libraries as small as possible. Tried to cover the most common usage first. Probably will supply a more extensive lib in a later version.

**Comment by Michael-F-Ellis** [comment]
Can we reopen this one? I very much need a strip() that does more than trim whitespace. If not, can you suggest a workaround that will do the right thing in both Cpython and Transcrypt?
Thanks!

**Comment by the1337guy** [comment]
Implementing this should be very easy, it's worth implementing.

**Comment by the1337guy** [comment]
I suggest we reopen this, too.

**Comment by Michael-F-Ellis** [comment]
The following works for my immediate problem but it makes my skin crawl a little.
```
                while a.startswith('-'):
                    a = a[1:]
                while a.endswith('-'):
                    a = a[:-1]
```

**Comment by fzzylogic** [comment]
Don't think Javascript has anything that does that natively, so if Transcrypt did do so, it would probably compile to something like like what you suggested, despite the discomfort that would induce :-). For [example](https://stackoverflow.com/questions/36390853/how-to-remove-specific-character-surrounding-a-string). The regex based [suggested polyfill](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/trim) for native trim() at MDN is also interesting.

**Comment by Michael-F-Ellis** [comment]
@fzzylogic Good point!  I think there are a couple of meta-issues here so I'm going to submit those separately to promote wider discussion.
